config/api/src/main/java/org/jboss/windup/config/metadata/LabelProviderData.java

```diff
@@ -0,0 +1,8 @@
+package org.jboss.windup.config.metadata;
+
+import java.util.List;
+
```

Comment:
Please add comments to describe the class and the methods as well.

Reply:
done